jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -78,7 +78,7 @@
 
 	private ShortPoint2D position;
 
-	private ShortPoint2D moveToRequest = null;
+	private ShortPoint2D targetPosition = null;
```

Comment:
We can also name it requestedTargetPosition.

It is a request by the user - not to be confused with the target the pathfinder uses (e.g. soldier attacking, ...).